examples/stimuli/video_playback.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+"""
+===============================================================================
+Script ''
+===============================================================================
+
+This script does XXX.
+"""
+# @author: drmccloy
+# Created on Tue Feb 23 13:57:53 2016
+# License: BSD (3-clause)
+
+from __future__ import print_function
+from expyfun import ExperimentController as EC
+
+movie_path = './example-movie.mp4'
```

Comment:
yeah, was planning to switch it to be handled like the HRTFs (DL from lester) before merging into master.